bundles/org.openhab.binding.livisismarthome/README.md

```diff
@@ -0,0 +1,215 @@
+
+However, a full example .things configuration look like this:
+
+```
+Bridge livisismarthome:bridge:mybride "LIVISI SmartHome Controller" {
+    Thing ISD2 myDimmer "Dimmer Kitchen" @ "Kitchen" [ id="<device-id>" ]
+    Thing ISS2 myLightSwitch "Light Livingroom" @ "Livingroom" [ id="<device-id>" ]
+    Thing PSS myTVSwitch "TV" @ "Livingroom" [ id="<device-id>" ]
+    Thing RST myHeating "Thermostat Livingroom" @ "Livingroom" [ id="<device-id>" ]
+    Thing ISR2 myRollerShutter1 "RollerShutter" @ "Livingroom" [ id="<device-id>" ]
+    Thing ISR2 myRollerShutter2 "RollerShutter (inverted)" @ "Livingroom" [ id="<device-id>" ] {Type RollerShutterActuator : rollershutter  [invert=true]}
+    Thing VariableActuator myLivisiVariable "My Variable" [ id="<device-id>" ]
+    Thing WDS myWindowContact "Window Kitchen" @ "Kitchen" [ id="<device-id>" ]
+    Thing WMD myMotionSensor "Motion entry" @ "Entry" [ id="<device-id>" ]
+    Thing WSC2 myPushButton "Pushbutton" @ "Living" [ id="<device-id>" ]
+    Thing WSD mySmokeDetector "Smoke detector Livingroom" @ "Living" [ id="<device-id>" ]
+}
+```
+
+## Items configuration
+
+You can then configure your items in your *.items config files as usual, for example:
+
+```
+Contact myWindowContact        "Kitchen"                <window>      {channel="livisismarthome:WDS:mybridge:myWindowContact:contact"}
+Switch myWindowContactBattery  "Battery low"            <battery>     {channel="livisismarthome:WDS:mybridge:myWindowContact:battery_low"}
+Number myHeatingTemp           "Bath [%.1f °C]"         <temperature> {channel="livisismarthome:RST:mybridge:myHeating:temperature"}
+Number myHeatingModeTempTarget "Settemp bath [%.1f °C]" <temperature> {channel="livisismarthome:RST:mybridge:myHeating:set_temperature"}
+String myHeatingMode           "Mode bath [%s]"         <temperature> {channel="livisismarthome:RST:mybridge:myHeating:operation_mode"}
+Number myHeatingHumidity       "Bath [%.1f %%]"         <humidity>    {channel="livisismarthome:RST:mybridge:myHeating:humidity"}
+
+```
+
+## Sitemap configuration
+
+The site configuration works a usual. One special example
+
+```
+sitemap default label="Home" {
+    Frame {
+        Text item=myHeatingTemp label="Temperature"
+        Text item=myHeatingHumidity label="Humidity"
+        Switch item=myHeatingMode label="Mode" mappings=[Manu="Manual", Auto="Auto"]
+        Setpoint item=myHeatingModeTempTarget label="Target temperature" minValue=16 maxValue=25 step=1
+    }
+}
+```
+
+## Rules example for push-buttons
+
+Push-buttons provide trigger channels, that can only be used in rules.
+Here is an example rule:
+
+```
+rule "Button triggered rule"
+when
+	Channel 'livisismarthome:WSC2:mybridge:myPushButton:button1' triggered PRESSED
+then
+    // do something...
+	logInfo("testlogger", "Button 1 pressed")
```

Comment:
Tab should be replaced by spaces for correct indentation:
```suggestion
    logInfo("testlogger", "Button 1 pressed")
```

Reply:
solved